ci-operator/step-registry/openshift/e2e/test/netobserv-qe/openshift-e2e-test-netobserv-qe-chain.yaml

```diff
@@ -22,8 +22,8 @@ chain:
   steps:
   - ref: idp-htpasswd
   - ref: cucushift-pre
-  - ref: openshift-extended-web-tests
   - ref: openshift-extended-test
+  - ref: openshift-extended-web-tests
```

Comment:
Reason for this change?

Reply:
because in last few runs when there openshift-extended-web-tests step fails, it doesn't get to openshift-extended-test. In past https://redhat-internal.slack.com/archives/C03SHCD5C9M/p1720723976144419  I had brought up similar behavior, however probably it also occurs when the openshift-extended-web-tests never exits in some cases (for us because of test were broken due to this PR https://github.com/openshift/openshift-tests-private/pull/20067) unless timeout is reached . So this change would have openshift-extended-test run first and then run openshift-extended-web-tests in the end